mpl_interactions/generic.py

```diff
@@ -306,18 +306,72 @@ class panhandler:
     """
     Enable panning a plot with any mouse button.
 
-    button determines which button will be used (default right click)
-    Left: 1
-    Middle: 2
-    Right: 3
+    .. code-block:: python
+
+       handler = panhandler(my_figure)
+
+       # Disable so it can be garbage collected
+       handler.disable()
```

Comment:
```suggestion
```
I actually don't think this is necessary. matlpotlib does something with weakrefs to make sure that this isn't a problem. This is why you need to keep a reference to keep it working

Reply:
Good point, fixed in f6689b8.